packages/pluggableWidgets/video-player-web/src/components/__tests__/SizeContainer.spec.tsx

```diff
@@ -0,0 +1,118 @@
+        const sizeContainer = create(<SizeContainer {...defaultProps} />).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit percentage of width", () => {
+        const sizeContainer = create(<SizeContainer {...defaultProps} heightUnit="percentageOfWidth" />).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with pixel width & height unit percentage of width", () => {
+        const sizeContainer = create(
+            <SizeContainer {...defaultProps} widthUnit="pixels" heightUnit="percentageOfWidth" />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit percentage of parent", () => {
+        const sizeContainer = create(<SizeContainer {...defaultProps} heightUnit="percentageOfParent" />).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit aspect ratio 1:1", () => {
+        const sizeContainer = create(
+            <SizeContainer {...defaultProps} heightUnit="aspectRatio" height={undefined} heightAspectRatio="oneByOne" />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit aspect ratio 4:3", () => {
+        const sizeContainer = create(
+            <SizeContainer
+                {...defaultProps}
+                heightUnit="aspectRatio"
+                height={undefined}
+                heightAspectRatio="fourByThree"
+            />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit aspect ratio 3:2", () => {
+        const sizeContainer = create(
+            <SizeContainer
+                {...defaultProps}
+                heightUnit="aspectRatio"
+                height={undefined}
+                heightAspectRatio="threeByTwo"
+            />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit aspect ratio 16:9", () => {
+        const sizeContainer = create(
+            <SizeContainer
+                {...defaultProps}
+                heightUnit="aspectRatio"
+                height={undefined}
+                heightAspectRatio="sixteenByNine"
+            />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with height unit aspect ratio 21:9", () => {
+        const sizeContainer = create(
+            <SizeContainer
+                {...defaultProps}
+                heightUnit="aspectRatio"
+                height={undefined}
+                heightAspectRatio="TwentyOneByNine"
+            />
+        ).toJSON();
+
+        expect(sizeContainer).toMatchSnapshot();
+    });
+
+    it("renders correctly with pixel width & height unit aspect ratio", () => {
```

Comment:
As mentioned [in my other comment](https://github.com/mendix/widgets-resources/pull/686#discussion_r661586974), can you now add one test with any of these aspect ratio and a percentage width that is not 100? So just 50% is fine. Otherwise I'm pretty sure we'll forget about these cases during future reviews or work